File: python/7 kyu/179.py
# https://www.codewars.com/kata/597d75744f4190857a00008d/train/python

# Passed

from collections import defaultdict

def paint_letterboxes(start, finish):
    frequency_dict = defaultdict(int)
    for num in range(start, finish + 1):
        for digit in str(num):
            frequency_dict[digit] += 1
                      
    frequency_list = [frequency_dict[str(digit)] for digit in range(10)]
    return frequency_list

output = paint_letterboxes(125, 132)
print(output)

File: nodejs/7 kyu/179.js
// https://www.codewars.com/kata/5631213916d70a0979000066/train/javascript

// Passed

function pattern(n) {
  let rows = [];
  for (let rowCount = 1; rowCount <= n; rowCount++) {
    let row =
      rowCount === 1 ? "1" : "1" + "*".repeat(rowCount - 1) + String(rowCount);
    rows.push(row);
  }

  let result = rows.join("\n");
  return result;
}

const output = pattern(20);
console.log(output);